web-api/openapi/back_office.yaml

```diff
@@ -32,9 +33,39 @@ info:
     Properties are optional data for documents, which are usually needed to properly show the document back to the user, when it returns as a personalized document.
     If for example, you'd wish to integrate a type of carousel view, listing a total of 10 personalized documents in a "for you"-section, then you might choose to display each document as an image and title, with a url to link the user to when pressed.
     For this, you would need three document properties: `image`, `link` and `title`.
+
+    ### Tags
+    Tags are optional data for documents, which are used to improve the scoring in document searches. Each document can have multiple tags.
+    For example, tags can be categories which the documents can be assigned to.
+  x-logo:
+    url: https://uploads-ssl.webflow.com/5ea197660b956f76d26f0026/637f7edd68c1ae2f12d4689e_Xayn%20Logo%202022%20-%20Footer.svg
+    altText: Xayn
 tags:
   - name: back office
     description: System management, including documents and properties.
+    x-traitTag: true
+  - name: documents
+    x-displayName: Documents
+  - name: candidates
+    x-displayName: Document candidates
+  - name: properties
+    x-displayName: Document properties
+  - name: property
+    x-displayName: Document property
+  - name: property indexing
+    x-displayName: Document property indexing
+x-tagGroups:
```

Comment:
these `x-` extensions are redocly specific, if we change the doc generator we need to adjust them accordingly.